crates/sui-indexer-alt-graphql/src/api/types/snapshots/sui_indexer_alt_graphql__api__types__available_range__field_piplines_tests__registry_collect_pipelines_snapshot.snap

```diff
@@ -0,0 +1,2505 @@
+---
+source: crates/sui-indexer-alt-graphql/src/api/types/available_range.rs
+expression: output
+---
+AccumulatorRootCreateTransaction._
+  => {}
+
+ActiveJwk.iss
+  => {}
+
+ActiveJwk.kid
+  => {}
+
+ActiveJwk.kty
+  => {}
+
+ActiveJwk.e
+  => {}
+
+ActiveJwk.n
+  => {}
+
+ActiveJwk.alg
+  => {}
+
+ActiveJwk.epoch
+  => {}
+
+ActiveJwkConnection.pageInfo
+  => {}
+
+ActiveJwkConnection.edges
+  => {}
+
+ActiveJwkConnection.nodes
+  => {}
+
+ActiveJwkEdge.node
+  => {}
+
+ActiveJwkEdge.cursor
+  => {}
+
+Address.address
+  => {}
+
+Address.asObject
+  => {"obj_versions"}
+
+Address.balance
+  => {"consistent"}
+
+Address.balances
+  => {"consistent"}
+
+Address.defaultSuinsName
+  => {"obj_versions"}
+
+Address.dynamicField
+  => {"obj_versions"}
+
+Address.dynamicFields
+  => {}
```

Comment:
Should include `consistent` here.

Reply:
Ugh, keep missing types and fields... removing the check to skip types on missing delegation entry for types that implement interfaces helped A LOT. 

It should be harder to miss entries now... I hope.